src/Hacks/antiaim.cpp

```diff
@@ -559,6 +559,37 @@ void DoAntiAimY(QAngle& angle, int command_number, bool bFlip, bool& clamp)
 			clamp = false;
 			angle.y = 36000180.0f;
 			break;
+		case AntiAimType_Y::LOWERBODY:
+			angle.y = *((C_BasePlayer*)entityList->GetClientEntity(engine->GetLocalPlayer()))->GetLowerBodyYawTarget() + rand()%35 + 165;
+			break;
+		case AntiAimType_Y::LBYONGROUND:
+			static C_BasePlayer* player = ((C_BasePlayer*) entityList->GetClientEntity(engine->GetLocalPlayer()));
+			if (player->GetFlags() & FL_ONGROUND)
+				angle.y = *((C_BasePlayer*)entityList->GetClientEntity(engine->GetLocalPlayer()))->GetLowerBodyYawTarget() + rand()%35 + 165;
+			else
+			{
+				static int aimType = rand() % 4;
```

Comment:
Is this supposed to be static?

rand() % 4 produces [0-4), do you want to cover condition 0?